docs/cheaha/migration_overview.md

```diff
@@ -0,0 +1,194 @@
+
+- **Monday, October 6**: Initial announcement concerning the migration is sent to the Cheaha userbase
+- **Wednesday, October 8**: Notification of impending migration sent to members of community 1
+- **Friday, October 10**: Migration for community 1 begins
+
+## GPFS 5 Data Tiering
+
+With the upgrade to GPFS 5, we are introducing a data tiering strategy to help us balance increasing storage demands with rising costs for high performance storage systems. GPFS will still be used as the primary parallel performance storage system and will store all files smaller than 4 MiB and all other active files. Inactive files will be transparently moved to a CephFS storage tier to reduce system pressure on GPFS. Inactivity will be determined by time since last file access. Files stored on CephFS leave a stub behind on GPFS. The stub contains all relevant metadata for the file and will appear to the user as if the file is still on GPFS. Reading a file stub will cause the file to be automatically transferred from CephFS back to GPFS.
+
+### What CephFS IS NOT
+
+1. **CephFS is NOT a replacement for LTS**.
+    1. Data you know you will not use for an extended period of time should be moved to [LTS](../data_management/lts/index.md).
+    1. Storage quotas will be enforced across both GPFS and CephFS. This means for a standard project quota, only 25 TiB can be stored in a project across both GPFS and Ceph.
+1. **CephFS is NOT a backup**
+    1. A file's data will only exist on either Ceph or GPFS, **never on both**.
+    1. If a file reaches the age limit, its data is automatically moved from GPFS to Ceph leaving behind a stub containing only file metadata.
+    1. When reading a file stub on GPFS, its data is moved from Ceph to GPFS leaving nothing on Ceph.
+1. **CephFS is NOT high performance storage for analysis**
+    1. Ceph only stores data deemed to be inactive on Cheaha.
+1. **CephFS is NOT something users will interact with**
+    1. Users will never make direct contact with Ceph. All of this is purely informational and provides background to some changes and behavior explained below.
+    1. All data transfer to and from Ceph is done transparently, without any direct action from the user. Do not change how you interact with Cheaha on account of this tiered storage.
+
+<!-- markdownlint-disable MD046 -->
+!!! critical
+    It is imperative to understand that tiered storage does not equal a backup. We do not provide a traditional, automatic backup for data stored on Cheaha. All data are erasure-encoded in case of hardware failure, they are not backed up in case of user error. Please see information about [LTS](../data_management/lts/index.md) for a potential backup solution.
+<!-- markdownlint-enable MD046 -->
+
+### What CephFS IS
```

Comment:
Put this before the "not" section, if possible. If you can, transform the "nots" into "is'es".

Reply:
Agreed, it's important to clarify boundaries and limitations. The more we can do so in a way has positive tone, the better our messaging comes across.